style/java.md

```diff
@@ -0,0 +1,33 @@
+# Java
+
+At Khan Academy, we only use Java for Android and, as such, follow the [Android Code Style Guidelines for Contributors](https://source.android.com/source/code-style.html). Note that those rules themselves extend the [standard Java language style guide as of 1999](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html). The rest of this document describes additions and clarifications to those rules that we follow at Khan Academy.
+
+
+## Imports
+
+Import statements are divided into the following groups, in this order, with each group separated by a blank line:
```

Comment:
Right - I was thinking that we'd do both codify and export the XML file to enforce it. You can also check it in with the project, so by default people cloning the main Android project would adopt it as well. But if we were to have multiple projects, anyone starting a new one would either have to know to import the XML to their global/personal settings or seed it or whatever.